miniupnpd/Makefile.linux_nft

```diff
@@ -117,7 +105,7 @@ install:	miniupnpd $(SRCDIR)/miniupnpd.8 $(SRCDIR)/miniupnpd.conf \
 	$(INSTALL) $(SRCDIR)/netfilter_nft/scripts/nft_delete_chain.sh $(DESTDIR)$(ETCINSTALLDIR)
 	$(INSTALL) $(SRCDIR)/netfilter_nft/scripts/miniupnpd_functions.sh $(DESTDIR)$(ETCINSTALLDIR)
 	$(INSTALL) -m 0644 -b $(SRCDIR)/miniupnpd.conf $(DESTDIR)$(ETCINSTALLDIR)
-	sed -i'' -e "s/^uuid=[-0-9a-f]*/uuid=$(UUID)/" $(DESTDIR)$(ETCINSTALLDIR)/miniupnpd.conf
+	sed -i -e "s/^uuid=[-0-9a-f]*/uuid=$(shell $(UUIDBIN))/" $(DESTDIR)$(ETCINSTALLDIR)/miniupnpd.conf
```

Comment:
7cd75e8f3ddeb72eb02b0cf239ccb5d976214290 `-i''` is on purpose !

Reply:
Then it needs to be a separate argument, no? Unless I'm fundamentally misunderstanding how BSDs work, the quote removal step runs in the shell and a `''` in the same word as `-i` makes absolutely no difference. To make this actually work as intended by 7cd75e8, there has to be a space between `-i` and `''`.

Reply:
Indeed I thing the space is necessary. I'm going to make a few tests with BSD sed

Reply:
Or perhaps just move `-i` to the end of the command line, in front of the file name?

Reply:
so as you noticed, `sed -i''` is the same as `sed -i` and works only with Gnu sed.
`sed -i ''` works with BSD sed, but not with Gnu sed. So I'll will continue to assume Linux uses GNU sed and *BSD BSD sed ;)

I guess I should use a variable `SED = sed -i`  / ` SED = sed -i ''` which could be overwritten for strange setups.

Reply:
@miniupnp like I suggested above, could you please test a `sed -e EXPR -i FILE` syntax?

Reply:
`sed: -I or -i may not be used with stdin`

Reply:
Oh. Yep I've finally taken a look at a BSD sed manpage, the argument after `-i` is the backup file extension, and not just a quirk as I originally thought. So yep it would need two versions of the command line.